pom.xml

```diff
@@ -24,9 +24,11 @@
         <tycho.scmUrl>scm:git:git://github.com:checkstyle/eclipse-cs.git</tycho.scmUrl>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.checkstyle.plugin.version>3.2.0</maven.checkstyle.plugin.version>
-        <maven.sevntu.checkstyle.plugin.version>1.42.0</maven.sevntu.checkstyle.plugin.version>
-        <!-- it should be a version of checkstyle that is compatible/compiled with sevntu -->
+        <!-- sevntu and patch-filters need to use a compatible/compiled version with checkstyle -->
         <maven.sevntu.checkstyle.plugin.checkstyle.version>10.0</maven.sevntu.checkstyle.plugin.checkstyle.version>
+        <maven.sevntu.checkstyle.plugin.version>1.42.0</maven.sevntu.checkstyle.plugin.version>
+        <maven.patch.filters.plugin.version>1.2.0</maven.patch.filters.plugin.version>
```

Comment:
Wording is updated to make it clear.
Moved all 3 dependencies so they are next to each other and grouped for visibility next to comment. 
Re-ordered properties to make more logical sense since all rely on checkstyle version.